src/pages/Exercise1/Exercise1.jsx

```diff
@@ -0,0 +1,21 @@
+import { useState } from "react";
+
+const Exercise1 = () => {
+  const [content, setContent] = useState("");
+  return (
+    <div>
+      <form>
+        <input
+          type="text"
+          value={content}
+          onChange={(event) => {
+            setContent(event.target.value);
+          }}
+        />
+      </form>
+      {content}
```

Comment:
Can you encapsulate this value in a `<p>` or similar element?

As it stands, the transpiled html would not be semantic